dashboard/src/components/TestDetails/TestDetails.tsx

```diff
@@ -66,70 +67,87 @@ const TestDetailsSection = ({ test }: { test: TTestDetails }): JSX.Element => {
 
   const hasUsefulLogInfo = test.log_url || test.log_excerpt;
 
-  const infos: ISubsection['infos'] = useMemo(() => {
-    const baseInfo: ISubsection['infos'] = [
-      {
-        title: 'global.status',
-        linkText: truncateBigText(test.status),
-      },
-      {
-        title: 'global.path',
-        linkText: valueOrEmpty(test.path),
-      },
-      {
-        title: 'global.arch',
-        linkText: valueOrEmpty(test.architecture),
-        icon: <PiComputerTowerThin className="text-blue" />,
-      },
-      {
-        title: 'global.compiler',
-        linkText: valueOrEmpty(test.compiler),
-      },
-      {
-        title: 'global.logs',
-        icon: hasUsefulLogInfo ? (
-          <MdFolderOpen className="text-blue" />
-        ) : undefined,
-        linkText: truncateBigText(valueOrEmpty(test.log_url)),
-        wrapperComponent: hasUsefulLogInfo ? SheetTrigger : undefined,
-      },
-      {
-        title: 'testDetails.gitCommitHash',
-        linkText: valueOrEmpty(test.git_commit_hash),
-      },
-      {
-        title: 'testDetails.gitRepositoryUrl',
-        linkText: truncateBigText(valueOrEmpty(test.git_repository_url)),
-        link: test.git_repository_url,
-      },
-      {
-        title: 'testDetails.gitRepositoryBranch',
-        linkText: valueOrEmpty(test.git_repository_branch),
-      },
-      {
-        title: 'testDetails.buildInfo',
-        linkText: truncateBigText(test.build_id),
-        linkComponent: buildDetailsLink,
-      },
-      {
-        title: 'global.hardware',
-        linkText: hardware,
-        icon: <GiFlatPlatform className="text-blue" />,
-      },
-      {
-        title: 'global.startTime',
-        linkText: (
-          <TooltipDateTime
-            dateTime={test.start_time}
-            lineBreak={true}
-            showLabelTime={true}
-            showLabelTZ={true}
-          />
-        ),
-      },
-    ];
-
-    return baseInfo;
+  const generalSection: ISection = useMemo(() => {
+    return {
+      title: test.path,
+      eyebrow: intl.formatMessage({ id: 'test.details' }),
```

Comment:
Since you are using `intl`only for the `formatMessage`function you can get it directly from the hook with 

```js
const { formatMessage } = useIntl();
```

and remove `intl` from the dependencies of your useMemo